cmake/projects/odb-compiler/hunter.cmake

```diff
@@ -29,9 +29,9 @@ elseif(CMAKE_HOST_UNIX)
       VERSION
       "2.4.0"
       URL
-      "http://www.codesynthesis.com/download/odb/2.4/odb-2.4.0-i686-linux-gnu.tar.bz2"
```

Comment:
where is `ARCHITECTURE_I686` set? I cannot find it in the docs nor in the source code (I jut grep'ed for it, though, didn't look any deeper) nor on Google.

On the other hand, I naively assume that most systems are 64bits nowadays, I just put the wrong compiler by mistake in the past because I wasn't compiling on Linux at the time.